mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -103,60 +102,6 @@ def load_model_with_mlflow_config(model_uri):
     return load_model(model_uri, **extra_kwargs)
 
 
-# Keep this method to maintain compatibility with MLServer
-# https://github.com/SeldonIO/MLServer/blob/caa173ab099a4ec002a7c252cbcc511646c261a6/runtimes/mlflow/mlserver_mlflow/runtime.py#L13C5-L13C31
-@deprecated("infer_and_parse_data", "2.6.0")
```

Comment:
Because of the above comment, we might not remove this... Maybe we should just remove the deprecated marker?

Reply:
You are right. Confirmed with @BenWilson2 , we should keep this method but remove the deprecation mark.